config/buildAssets.js

```diff
@@ -67,22 +77,20 @@ esbuild
     minify: process.env.ELEVENTY_ENV === 'production',
     sourcemap: process.env.ELEVENTY_ENV !== 'production',
     target: ['chrome58', 'firefox57', 'safari11', 'edge18'],
-    plugins: [ 
+    plugins: [
       sassPlugin({
         loadPaths: [
-          "./node_modules/@uswds",
-          "./node_modules/@uswds/uswds/packages"
-        ]
+          './node_modules/@uswds',
+          './node_modules/@uswds/uswds/packages',
+        ],
       }),
     ],
     bundle: true,
   })
   .then(() => createAssetPaths())
   .then(() => {
-    console.log('Assets have been built!');
```

Comment:
Might we still want some sort of logging of the build step?

Reply:
For sure, fixed in 81882fb